### ISSUE TITLE
Feature request: Can you add "binaural" to the target speaker layout options?

### ISSUE THREAD

**Issue by bluewidy** [ittiam-systems/libmpegh#63]

**Comment by bluewidy** [comment]
Binaural refers to a layout where you can feel the virtualization of 3D audio on your headphones. 
So what I expect is what is in this screenshot. The MPEG-H VVplayer provided by Fraunhofer already provides binaural for the target speaker layout. I hope you can hack this.
<img width="776" alt="MPEG-H VVPlayer" src="https://github.com/ittiam-systems/libmpegh/assets/40682277/c4606f67-0f02-4057-80ce-b8b43585b015">

**Comment by bluewidy** [comment]
And I tried the brir parameter with my brir file and it doesn't work.
Here is the brir file I used.
[bluewindys_IRs_for_7.1.4.zip](https://github.com/ittiam-systems/libmpegh/files/12194801/bluewindys_IRs_for_7.1.4.zip)

**Comment by SakethSathuvalli** [comment]
Can You share the command line ?

**Comment by bluewidy** [comment]
`ia_mpeghd_testbench.exe -ifile:sine_1khz_cicp19.mp4 -ibrir:bluewindys_IRs_for_7.1.4.wav -ofile:result.wav`

**Comment by SakethSathuvalli** [comment]
> `ia_mpeghd_testbench.exe -ifile:sine_1khz_cicp19.mp4 -ibrir:bluewindys_IRs_for_7.1.4.wav -ofile:result.wav`

Can You please confirm if the file - `bluewindys_IRs_for_7.1.4.wav` - complies with the bitstream format as suggested in section 17.4.2 of [MPEG-H 3D Audio specification document](https://www.iso.org/standard/83525.html)?

**Comment by bluewidy** [comment]
I succeeded in reading the documentation via a different route. So if I understand correctly, I need to provide the directory containing the multiple BRIR files as an argument to the parameter, right?

**Comment by SakethSathuvalli** [comment]
> I succeeded in reading the documentation via a different route. So if I understand correctly, I need to provide the directory containing the multiple BRIR files as an argument to the parameter, right?

Yes, You need to provide the path of the BRIR file (that complies) with the format specified by the MPEG-H 3DA specification document as parameter.

**Comment by SakethSathuvalli** [comment]
> And I tried the brir parameter with my brir file and it doesn't work. Here is the brir file I used. [bluewindys_IRs_for_7.1.4.zip](https://github.com/ittiam-systems/libmpegh/files/12194801/bluewindys_IRs_for_7.1.4.zip)

We will look into the possibility of a tool to convert this into a bitstream format that MPEG-H decoder expects.

**Comment by bluewidy** [comment]
Thank you
